SWLOR.Game.Server/Perk/ForceAlter/ForcePush.cs

```diff
@@ -106,6 +106,7 @@ public void OnImpact(NWCreature creature, NWObject target, int perkLevel, int sp
                 }
 
                 _.ApplyEffectToObject(DurationType.Temporary, AbilityService.EffectKnockdown(target, duration), target, duration);
+                _.ApplyEffectToObject(DurationType.Temporary, _.EffectSlow(), target, duration);
```

Comment:
What's the purpose of doing both a knockdown and a slow?

Reply:
That way it still has a effect on people with balance stance up, push is really weird in the fact is incredibly broken or just useless because of balanced stance.